examples/docking-protein-protein/docking-protein-protein-cltsel.cfg

```diff
@@ -0,0 +1,64 @@
+# ====================================================================
+# Rigid-body docking example
+
+# directory in which the scoring will be done
+run_dir = "run1-clt"
+ncores = 20
+# cns_exec = "path/to/bin/cns" # optional
+
+# molecules to be docked
+molecules =  [
+    "data/e2aP_1F3G.pdb",
+    "data/hpr_ensemble.pdb"
+    ]
+
+# ====================================================================
+# Parameters for each stage are defined below, prefer full paths
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
+#####################################################################
+[topoaa]
+autohis = false
+[topoaa.input.mol1]
+nhisd = 0
+nhise = 1
+hise_1 = 75
+[topoaa.input.mol2]
+nhisd = 1
+hisd_1 = 76
+nhise = 1
+hise_1 = 15
+
+[rigidbody]
+ambig_fname = 'data/e2a-hpr_air.tbl'
+sampling = 100
+noecv = true
+
+[clustfcc]
+
+[seletopclusts]
+# select the best 4 clusters
+top_cluster = [1, 2, 3, 4]
+# select the best 4 models of each cluster
+top_models = 4
+
+[caprieval]
+reference_fname = 'data/e2a-hpr_1GGR.pdb'
+
+[flexref]
+ambig_fname = 'data/e2a-hpr_air.tbl'
+noecv = true
+
+[caprieval]
+reference_fname = 'data/e2a-hpr_1GGR.pdb'
+
+[emref]
+ambig_fname = 'data/e2a-hpr_air.tbl'
+noecv = true
+
+[caprieval]
+reference_fname = 'data/e2a-hpr_1GGR.pdb'
+
+# ====================================================================
```

Comment:
While at it - since in HADDOCK our scoring scheme should be by default cluster-based scoring - I would add a step to perform the final clustering and scoring (can `caprieval` work on a cluster-basis?)

Reply:
`caprieval` works if you put it after clustering :D 

In this `cltsel` example the workflow is: 
`topoaa > rigidbody > clustfcc > seletopclusts > caprieval > flexref > caprieval > emref`

You mean we should add another `clustfcc` and `emscoring` in the end?
`topoaa > rigidbody > clustfcc > seletopclusts > caprieval > flexref > caprieval > emref > clustfcc > emscoring > caprieval`

Or that we should add it to the main protein-protein example?